cibuild.cmd

```diff
@@ -102,6 +102,7 @@ if defined TestPerfRun (
 
             REM Get the benchview tools - Place alongside Roslyn.Test.Performance.Runner.exe
             call "%RoslynRoot%\build\scripts\install_benchview_tools.cmd" ".\Binaries\%BuildConfiguration%\" || goto :BuildFailed
+            dir ".\Binaries\%BuildConfiguration%\"
```

Comment:
I'm still debugging this... Apparently if you try to do a `nuget install` into a folder that doesn't exist, nuget will say `Unable to find package 'Microsoft.BenchView.JSONFormat'`.